src/Common/src/Internal/NativeFormat/NativeFormatWriter.cs

```diff
@@ -1658,12 +1658,24 @@ class VertexTree : Vertex
             private Vertex _first;
             private Vertex _second;
 
+            public VertexTree()
+            {
+                _first = null;
+                _second = null;
+            }
+
```

Comment:
I actually later fixed one more bug in the VertexArray in commit

https://github.com/trylek/corert/commit/f5cf18ce1f05d393a54b2234edf1c0d27c7c5d7a

Please feel free to decide whether you'd prefer to cherrypick that from my commit and add to your change or deal with later.

Thanks a lot for doing this!

Tomas